fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -614,4 +617,37 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    protected void createFixityEvent(final EventBus eventBus, final RdfStream rs,
```

Comment:
Why is this method `void` with the event bus as a parameter? Why not have it return the assembled `FixityEvent` and let the caller manage its publication to the event bus? That would seem clearer.